src/map/trade.cpp

```diff
@@ -362,12 +362,10 @@ void trade_tradeadditem(struct map_session_data *sd, short index, short amount)
 	}
 
 	if( !amount ) { // Why do this.. ~.~ just send an ack, the item won't display on the trade window.
-		clif_tradeitemok(sd, index, 0);
+		clif_tradeitemok(*sd, index, EXITEM_ADD_SUCCEED);
```

Comment:
Since we removed the custom response below, we should maybe remove that one (which happens before the index validation) too?

Reply:
Official server sends index 0, and amount 0 in this case.